File: Patterns/pattern8.cpp
#include <iostream>
using namespace std;

int main(){
	int n;
	cin >> n;

	int count = 1;
	for( int i = 0; i < n; i++ ){
		for( int j = 0; j <= i; j++ ){
			if ( n < count){
				return 0;
			}
			cout << count << " ";
			count++;
		}
		cout << endl;
	}
	cout << endl;
    return 0;

}

/*
10 

1
23
456
78910

*/

File: Patterns/pattern8.py
#This is implementation of pattern8 in python3
# Input n as integer
n = int(input())

#Set count =1
count = 1

# for loop for printing the patterns
for i in range(0,n):
    s=""
    for j in range(0,i+1):
        if n >= count:
            s=s+str(count)
            count+=1
    print(s)


##Output
##10
##1
##23
##456
##78910

